terraform/digitalocean/project/terraform.tf

```diff
@@ -3,12 +3,12 @@ terraform {
   required_version = ">= 0.12.0"
 
   backend "s3" {
-    region = "us-east-1"
-    access_key = ""
-    secret_key = ""
-    bucket = ""
-    key = ""
-    endpoint = ""
+    region                      = "us-east-1"
+    access_key                  = ""
+    secret_key                  = ""
+    bucket                      = ""
+    key                         = ""
+    endpoint                    = ""
```

Comment:
[p5]
```suggestion
    region     = "us-east-1"
    access_key = ""
    secret_key = ""
    bucket     = ""
    key        = ""
    endpoint   = ""

```

Reply:
I have removed project. Since it is not used now